webpack/assets/javascripts/foreman_hosts.js

```diff
@@ -1,4 +1,13 @@
 import { uniq, set } from 'lodash';
+import * as table from './hosts/tableCheckboxes';
+
+export default {
```

Comment:
Not sure if this is preferred way of exporting, maybe it is - it is stating at the beginning all the exports. but I would deleted the other exports than - why to export same stuff twice? :thinking: